frontend/src/utils/helpers.ts

```diff
@@ -169,3 +175,45 @@ export const displayDateInfo = (date: Date) => {
 export const formatRoleOrStatus = (str: string) => {
   return str[0] + str.substring(1).toLowerCase();
 };
+
+/**
+ * Upload image to firebase
+ * @param userID          - The userID of who uploads the image
+ * @param image           - The image file to upload
+ * @returns The string URL to the event.
+ */
+export const uploadImage = async (userID: string | null, image: File) => {
+  if (!userID) {
+    console.error("No eventId provided");
```

Comment:
Hmmm how come you are checking if no userID is passed but it errors out as no eventid provided

Reply:
Ah sorry, I kept the console error from before we decided to switch to user ID to name the image.